terraform/team-members-datadog/_terraform.tf

```diff
@@ -0,0 +1,22 @@
+// Configuration for Terraform itself.
+
+terraform {
+  required_version = "~> 1"
+
+  required_providers {
+    datadog = {
+      source  = "datadog/datadog"
+      version = "3.24.1"
+    }
+  }
+
+  backend "s3" {
+    bucket         = "rust-terraform"
+    key            = "simpleinfra/team-members-datadog.tfstate"
+    region         = "us-west-1"
+    dynamodb_table = "terraform-state-lock"
+    encrypt        = true
+  }
+}
```

Comment:
I wander, doesn't `simpleinfra` use Terragrunt? Wouldn't it be better to use the DRY it offers?

Reply:
Terragrunt is really handy for things that have multiple environments like `production` and `staging`. The team access will only ever have a single instance, so we haven't bothered moving it over to Terragrunt yet. And since the other rest of the team access is managed with Terraform as well, I didn't want to make this a one-off configuration in a different place.